interfaces/HO-Portal/src/app/models/user.model.ts

```diff
@@ -1,11 +1,9 @@
-import Permission from '../auth/permission.enum';
-
 export class UserModel {
   token: string;
 }
 
 export class User {
   token?: string;
   username: string;
-  permissions: Permission[];
+  permissions: string[];
```

Comment:
The type `Permission` in the interface(s) is not the same as the `Permission`-object in the back-end;
So we should keep this as it was.